### DISCUSSION TITLE
Idea for addition to documentation

### DISCUSSION THREAD

**Discussion by jensgreven** [SillyTavern/SillyTavern#3202] [Ideas]
On the model card for [Mixtral-8x7B-Instruct-v0.1-LimaRP-ZLoss-6.0bpw-h6-exl2](https://huggingface.co/LoneStriker/Mixtral-8x7B-Instruct-v0.1-LimaRP-ZLoss-6.0bpw-h6-exl2) I read about the feature for directing the model's answer length:

> Message length control
>
> Due to the inclusion of LimaRP v3, it is possible to append a length modifier to the response instruction sequence, like this:
> 
> ### Input
> User: {utterance}
> 
> ### Response: (length = medium)
> Character: {utterance}
>
> This has an immediately noticeable effect on bot responses. The available lengths are: micro, tiny, short, medium, long, 
> massive, huge, enormous, humongous, unlimited. The recommended starting length is medium. Keep in mind that the AI may 
> ramble or impersonate the user with very long messages.
>
> Message length control
> 
> Due to the inclusion of LimaRP v3, it is possible to append a length modifier to the response instruction sequence, like this:
>
> ### Input
> User: {utterance}
> 
> ### Response: (length = medium)
> Character: {utterance}
> 
> This has an immediately noticeable effect on bot responses. The available lengths are: micro, tiny, short, medium, long, 
> massive, huge, enormous, humongous, unlimited. The recommended starting length is medium. Keep in mind that the AI may 
> ramble or impersonate the user with very long messages.

Maybe it's worth adding that to the ST documentation?

**Comment by Cohee1207** [comment]
Model-specific prompt configuration is not an expected to be added into documentation.
